code/game/objects/items/devices/laserpointer.dm

```diff
@@ -197,7 +197,8 @@
 	if(!ISADVANCEDTOOLUSER(user))
 		to_chat(user, span_warning("You don't have the dexterity to do this!"))
 		return
-	if(HAS_TRAIT(user, TRAIT_CHUNKYFINGERS))
+	var/mob/living/carbon/human/human_user = user
+	if(human_user.check_chunky_fingers())
```

Comment:
Instead of casting them to human (very unlikely but possible that they are not), move this proc definition to `living` with the more basic trait check in it.

Reply:
feel like adding the /lving/ proc definition as a simpler version is out of scope and only really necessary when we add a simplemob that uses chunky_fingers, since all current implementations of the check use the human version anyways

will cope and write it if you point me to the file to add this in, maybe mob/mob_helpers, but there's no file for it in living iirc